### PR TITLE
feat: adding breadcrumbs on My Account page

### PR DESCRIPTION
**Pull request by scernisevs**
adding breadcrumbs on My Account page
## Description
adding breadcrumbs on My Account page

## Related Issue
VBC-404

## Motivation and Context
There were no breadcrumbs of the active page in My Account, also 'My Account' crumb was directing user to the Home page.

## How Has This Been Tested?
Locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [X] I have read the **CONTRIBUTING** document.
- [X] All new and existing tests passed.

### CODE REVIEW

File: packages/theme/pages/MyAccount.vue [hunk 1]
```diff
@@ -77,6 +77,17 @@ export default {
         return '';
       }
     });
+    const breadcrumbs = ref([
+      {
+        text: 'Home',
```
Comment (WojtekTheWebDev) [inline packages/theme/pages/MyAccount.vue:82]: Shouldn't this text be translated?

File: packages/theme/pages/MyAccount.vue [hunk 2]
```diff
@@ -77,6 +77,17 @@ export default {
...
+        link: '/'
+      },
+      {
+        text: 'My Account',
```
Comment (WojtekTheWebDev) [inline packages/theme/pages/MyAccount.vue:86]: The same as above

**Review by WojtekTheWebDev — CHANGES_REQUESTED** [review]
Just one question in the comments

**Review by dhargitai — APPROVED** [review]
LGTM :)

**Review by WojtekTheWebDev — APPROVED** [review]
LGTM

File: packages/theme/pages/MyAccount.vue (hunk 1 above)
Comment (scernisevs) [reply packages/theme/pages/MyAccount.vue]: thank you for spotting this! translation added

**Pull request merged**